roles/openshift_examples/files/examples/x86_64/xpaas-streams/eap71-image-stream.json

```diff
@@ -17,11 +17,11 @@
                 "annotations": {
                     "openshift.io/display-name": "Red Hat JBoss EAP 7.1",
                     "openshift.io/provider-display-name": "Red Hat, Inc.",
-                    "version": "1.4.14"
+                    "version": "1.4.17"
```

Comment:
@luck3y do you know why this is 1.4.17 rather than 1.4.18? We're trying to get everything back into shape where it's exclusively managed via sync script so we've bumped `XPAAS_VERSION=ose-v1.4.18` in examples-sync.sh and this is one of the diffs we noticed, as well as most other files where your PR bumps to 1.4.17.

Reply:
@sdodson Hmm, I thought we'd fix this already, I'll take a look at the source, fix if needed and report back.

Reply:
@sdodson This looks correct in the upstream tag ose-v1.4.18:

https://github.com/jboss-openshift/application-templates/blob/ose-v1.4.18/eap/eap70-image-stream.json#L20

I'll update this PR